components/docs-chef-io/content/automate/ha_cert_rotation.md

```diff
@@ -110,41 +112,40 @@ You can also use `--cc` instead of `--certificate-config` as a sort form.
         ip = "10.1.2.247"
         public_key = "full path of os3.pem"
         private_key = "full path of os3-key.pem"
-    ```
-
+```
 
-### Rotate Certificates of each service
+### Rotate Certificates of Each Service
 
 If you want to rotate certificates of the entire cluster, then you can follow the below commands:
 
 - To rotate certificates of automate cluster:
 
-    ```cmd 
-    chef-automate cert-rotate --public-cert <path of public certificate of automate node> --private-cert <path of private certificate of automate node> --a2
+    ```cmd
```

Comment:
`cmd` isn't a proper format. These should be `sh` or `bash`.
And, like above, code sample text should be aligned with the tick marks.